cpp/oneapi/dal/algo/pca/test/batch.cpp

```diff
@@ -194,10 +194,13 @@ using pca_types = COMBINE_TYPES((float, double), (pca::method::cov, pca::method:
 
 TEMPLATE_LIST_TEST_M(pca_batch_test, "pca common flow", "[pca][integration][batch]", pca_types) {
     SKIP_IF(this->not_available_on_device());
+    
+    const std::string higgs = "/higgs/dataset/higgs_100t_train.csv";
```

Comment:
```suggestion
    const std::string higgs = "higgs/dataset/higgs_100t_train.csv";
```

Relative dataset path must be specified without "/" in the beginning.